sites/all/modules/hr/hr_disasters/hr_disasters.module

```diff
@@ -66,12 +66,11 @@ function _hr_disasters_reliefweb_disaster($id) {
         ->addTag('DANGEROUS_ACCESS_CHECK_OPT_OUT')
         ->execute();
       if (!empty($results['node'])) {
-        // HACK to keep this disaster not in sync with Reliefweb, since we "extended" it to multiple countries
-        if ($glide != 'EP-2014-000116-GHA' && $glide != 'OT-2011-000205-NER' && $glide != 'EP-2014-000041-GIN') {
-          // Disaster already exists, simply update
-          $nids = array_keys($results['node']);
-          $nid = reset($nids);
-          $node = node_load($nid);
+        $nids = array_keys($results['node']);
+        $nid = reset($nids);
+        $wrapper = entity_metadata_wrapper('node', $nid);
+        $field = $wrapper->field_hr_disaster_sync_disaster->raw();
```

Comment:
please, use value() instead of raw().
There's also the chance that this field might not be informed in the entity and then the code will fail, you need to add checks for that, see http://drupal.stackexchange.com/questions/94748/checking-for-existence-for-a-field-on-an-an-entity-metadata-wrapper
Also, $field doesn't look like a good variable name.